README.md

```diff
@@ -370,35 +370,19 @@ restApiConfig:
 
 ## Step 9: Customize Model Deployment
 
-In the `ecsModels` section of `config.yaml`, configure the models you want to deploy.
+In the `ecsModels` section of `config.yaml`, allow our deployment process to pull the model weights for you.
 
-The configuration file will determine which models are deployed. In order to deploy an additional model or a
-different model the only required change is to the configuration file, as long as it is compatible with the inference
-container. Specifically, see the `ecsModels` section of the [config.yaml](./config.yaml) file.
-Here we define the model name, if we want to deploy, the type of instance we want to deploy to, the type of model
-(textgen or embedding), the inference container and then the containerConfig. There are many more parameters for the
-ecs models, many for autoscaling and health checks. However, let's focus on the model specific ones:
+During the deployment proces, lisa will optionally attempt to download your model weights if you specify an optional`ecsModels`
+array, this will only work in non ADC regions. Specifically, see the `ecsModels` section of the [config.yaml](./config.yaml) file.
+Here we define the model name, inference container, and baseImage:
 
 ```yaml
 ecsModels:
   - modelName: your-model-name
-    deploy: false
-    instanceType: g4dn.12xlarge
-    modelType: textgen
     inferenceContainer: tgi
```

Comment:
we can simplify even further by removing this portion from the config. we'd only need to check for the docker image since the tgi/tei/vllm options only matter for the path component at deployment time, but not for artifact collection time. all we would need are the model name (`$provider/$model` format) and the docker image/tag that we'd like to run with. even with the docker image- we don't do any validations on that initially that I'm aware of, and we just *assume* that the customer has access to the image at deployment time (during createmodel). so ultimately, this could just be a list of modelNames in huggingface $provider/$model format

Reply:
It was using the docker image and inference container in this section of the make file:

```
						echo "Converting and uploading safetensors for model: $(MODEL_ID)"; \
						tgiImage=$$(yq -r '[.$(ENV).ecsModels[] | select(.inferenceContainer == "tgi") | .baseImage] | first' $(PROJECT_DIR)/config.yaml); \
						echo $$tgiImage; \
						$(PROJECT_DIR)/scripts/convert-and-upload-model.sh -m $(MODEL_ID) -s $(MODEL_BUCKET) -a $$access_token -t $$tgiImage -d $$localModelDir; \
```

so rather than doing an entire refactor I was trying to keep the changes limited. I can see what the convert and upload script uses them for.